src/ReactiveUI.XamForms.Tests/ReactiveUI.XamForms.Tests.csproj

```diff
@@ -9,4 +9,34 @@
     <ProjectReference Include="..\ReactiveUI.XamForms\ReactiveUI.XamForms.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
If these references aren't needed, then can we remove them and keep the project files as light as possible?

Reply:
Removed